model/src/commonMain/kotlin/io/github/droidkaigi/feeder/TimetableItem.kt

```diff
@@ -62,4 +64,9 @@ sealed class TimetableItem(
         asset = asset,
         levels = levels,
     )
+
+    val startsTimeString: String by lazy {
+        val localDate = startsAt.toLocalDateTime(TimeZone.currentSystemDefault())
+        "${localDate.hour}".padStart(2,'0') +  ":" + "${localDate.minute}".padStart(2,'0')
```

Comment:
Can you apply format? 🙏 

```suggestion
        "${localDate.hour}".padStart(2, '0') + ":" + "${localDate.minute}".padStart(2, '0')
```